src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +85,12 @@ public String toString() {
         return getAsTextShowAll();
     }
 
-}
+    //returns a concatenated details of a person
+    public String getPrintableString(Printable... printables) {
+         String toPrint =""; 
+         for(Printable toAdd: printables) {
+             toPrint+= toAdd.getPrintableString()+", ";
```

Comment:
- Java has a `StringJoiner` class that can be used to join strings.
- There must be an empty space in-between symbols (`+=`, `+`) and after `for`